CHANGELOG.md

```diff
@@ -2012,10 +2012,10 @@ current changes on git with [previous release tags][git_tag_comparison].
 - [Add convenient methods for Image][10221]
 - [Use “specular occlusion” term to consistently extinguish fresnel on Ambient and Environment Map lights][10182]
 - [Fix fog color being inaccurate][10226]
-- [Replace all usages of texture_descritor.size.* with the helper methods][10227]
+- [Replace all usages of texture_descritor.size.\* with the helper methods][10227]
 - [View Transformations][9726]
 - [fix deferred example fog values][10249]
-- [WebGL2: fix import path for unpack_unorm3x4_plus_unorm_20_][10251]
+- [WebGL2: fix import path for unpack*unorm3x4_plus_unorm_20*][10251]
```

Comment:
there's a lot of spurious changes here, can you try to make the diff minimal? I dont think the past changelog items should be changed.

Reply:
Yeah I did a find and replace and for some reason my editor decided to format these :P should be fixed now